common_states/env-hooks/common_states.sh

```diff
@@ -1 +1,3 @@
-export RAFCON_LIBRARY_PATH=$(rospack find common_states)/states/common_states:$RAFCON_LIBRARY_PATH
\ No newline at end of file
+if [[ "$RAFCON_LIBRARY_PATH" != *":$(rospack find common_states)/states/common_states:"* ]]; then
```

Comment:
This is working in zsh but producing a lot of output during builds now. I'll look to see if there's a cross shell way to fix that